### DISCUSSION TITLE
Is [read_def -floorplan_initialize ] expected to unset [set_max_transition] and [set_max_capacitance] from sdc file?

### DISCUSSION THREAD

**Discussion by b62833** [The-OpenROAD-Project/OpenROAD#4845] [General]
Before I file a bug, I want to make sure this is unexpected behavior.

In my sdc file I do a 
```

set_max_transition [exp $clk_period * 0.20] [current_design]
set_max_capacitance 30 [current_design]
```

and then read a floorplan by setting FLOORPLAN_DEF.

When I run through floorplanning, 2_floorplan.sdc doesn't have those commands. I've narrowed it down to floorplan.tcl's `read_def -floorplan_initialize` that's removing these. Max transition and max capacitance can't be set in DEF if I remember correctly, so this is unexpected. Is this the desired tool behavior?

**Comment by maliberty** [comment]
I would consider it unexpected so go ahead and open an issue with a test case

**Comment by b62833** [comment]
Over the weekend I upgraded to OpenROAD version 2.0-13286-gebcc5196e and it works now.

It was broken in the version v2.0-11493-gaddecc2bd that I'd been using before.

We're good now.
